perses/analysis/load_simulations.py

```diff
@@ -2,55 +2,69 @@
 import matplotlib.pyplot as plt
 import sys
 from openeye import oechem, oegraphsim
-import logging
-
-
-_logger = logging.getLogger("analysis")
-
-
-class Molecule(object):
-    def __init__(self, i, string):
-        from perses.utils.openeye import smiles_to_oemol
-        self.line = string
-        details = string.split(';')
-        self.index = i
-        self.smiles, self.name, self.exp, self.experr, self.calc, self.calcerr = details[1:7]
-        self.mol = smiles_to_oemol(self.smiles)
-        self.exp = kcal_to_kt(float(self.exp))
-        self.experr = kcal_to_kt(float(self.experr))
-        self.calc = kcal_to_kt(float(self.calc))
-        self.calcerr = kcal_to_kt(float(self.calcerr))
-        self.mw = self.calculate_molecular_weight()
-        self.ha = self.heavy_atom_count()
-        self.simtype = None
-
-    def calculate_molecular_weight(self):
-        """ Calculates the molecular weight of an oemol
-
-        Parameters
-        ----------
 
-        Returns
-        -------
-        float, molecular weight of molecule
-
-        """
-        return oechem.OECalculateMolecularWeight(self.mol)
+import logging
+_logger = logging.getLogger()
+_logger.setLevel(logging.INFO)
+_logger = logging.getLogger("load_simulations")
 
-    def heavy_atom_count(self):
-        """ Counts the number of heavy atoms in an oemol
 
-        Parameters
-        ----------
+class Simulation(object):
+    """Object that holds the results of a free energy simulation.
```

Comment:
Can you note that this automatically loads the data and performs the analysis, making the results available through attributes below?

A quick docstring example would be great:
```python
>>> # Load and analyze the simulation for `lig0to1/out.*`
>>> sim = Simulation(0, 1)
>>> # Report the binding free energy
>>> print(f'Binding free energy = {sim.bindingdg} +- {sim.bindingddg}')
```